docs/user-guide/release-notes/master.rst

```diff
@@ -55,6 +55,22 @@ Known Issues
 Upgrade Instructions for 2.6.x --> master
 -----------------------------------------
 
+Upgrade the packages using:
+:::
```

Comment:
I recommended the wrong fix in my earlier review. This doesn't compile correctly either. Try using `::` throughout you changes like this:

```
Upgrade the packages using::

    sudo yum update
```